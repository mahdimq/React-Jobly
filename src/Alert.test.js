import React from 'react';
import { render } from '@testing-library/react';
import Alert from './Alert';

it('renders without crashing', function () {
	render(<Alert />);
});

it('matches snapshot for error', function () {
	let messages = ['It does not work!'];
	const { asFragment } = render(<Alert messages={messages} />);
	expect(asFragment()).toMatchSnapshot();
});

it('matches snapshot for no errors', function () {
	let messages = ['It works!'];
	const { asFragment } = render(<Alert messages={messages} />);
	expect(asFragment()).toMatchSnapshot();
});
