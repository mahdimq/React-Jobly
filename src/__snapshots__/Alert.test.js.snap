// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot for error 1`] = `
<DocumentFragment>
  <div
    class="alert"
  >
    <p>
      It does not work!
    </p>
  </div>
</DocumentFragment>
`;

exports[`matches snapshot for no errors 1`] = `
<DocumentFragment>
  <div
    class="alert"
  >
    <p>
      It works!
    </p>
  </div>
</DocumentFragment>
`;
